src/index.html

```diff
@@ -8,6 +8,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars--0">
+        <img src="images/star.svg" alt="star">
+        <img src="images/star.svg" alt="star">
+        <img src="images/star.svg" alt="star">
+        <img src="images/star.svg" alt="star">
+        <img src="images/star.svg" alt="star">
+      </div>
```

Comment:
```suggestion
    <div class="stars-block stars--0">
      <div class="star"></div>
      <div class="star"></div>
      <div class="star"></div>
      <div class="star"></div>
      <div class="star"></div>
    </div>
```

давай зробим так щоб працював ось такий запис, а картинки будем задавати як фон